src/Utils.ps1

```diff
@@ -280,21 +280,36 @@ function Get-PathStringComparison {
 
 function Get-PromptPath {
     $settings = $global:GitPromptSettings
-    $abbrevHomeDir = $settings -and $settings.DefaultPromptAbbreviateHomeDirectory
+    $stringComparison = Get-PathStringComparison
 
     # A UNC path has no drive so it's better to use the ProviderPath e.g. "\\server\share".
     # However for any path with a drive defined, it's better to use the Path property.
     # In this case, ProviderPath is "\LocalMachine\My"" whereas Path is "Cert:\LocalMachine\My".
     # The latter is more desirable.
     $pathInfo = $ExecutionContext.SessionState.Path.CurrentLocation
     $currentPath = if ($pathInfo.Drive) { $pathInfo.Path } else { $pathInfo.ProviderPath }
+    if (!$settings -or !$currentPath -or $currentPath.Equals($Home, $stringComparison)) {
+        return $currentPath
+    }
+    $abbrevHomeDir = $settings.DefaultPromptAbbreviateHomeDirectory
```

Comment:
Extremely minor nit, sorry but could you add a blank line above this one to separate the previous `if` statement from this set of variable assignments.